### DISCUSSION TITLE
Pandas support?

### DISCUSSION THREAD

**Discussion by plaksnor** [OpenLineage/OpenLineage#2034] [Q&A] [answered]
Hi all,
I'm using OL in Azure Synapse Notebooks which works great.
There are a lot of Spark events written to the logs, but I'm missing some Pandas activities.
Since our users are able to use any type of Pandas functionality I was wondering if Pandas is supported.

For example, [pandas.DataFrame.to_csv](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html).

Is there a page where I can see what kind of functionality is supported or not?
Thanks in advance!

**Comment by mobuchowski** [comment, marked as answer]
Pandas is not yet supported, but this has been one of the most popular requests. I'm sure there will be more on that topic in some time.
